### ISSUE TITLE
[Improvement] Catch exception for missing Google-Service files

### ISSUE THREAD

**Issue by phihochzwei** [distriqt/ANE-Firebase#46]
If you forget to include the google-service.json or the GoogleService-Info.plist (depending on your target), the apps will crash when you call initialiseApp(). Apple tells you 

`<Notice>: *** Terminating app due to uncaught exception 'com.firebase.core', reason: '[FIRApp configure] could not find a valid GoogleService-Info.plist in your project. Please download one from https://console.firebase.google.com/.'`

I think it would be great to have this exception caught, so that basically the app is working but in the terminal you get a message, that the firebase services are not available. 

Am I making any sense here xD ?

**Comment by marchbold** [comment]
Yeah definitely making sense, we aren't sure that is possible though. 

We already are catching any exceptions thrown by the `[FIRApp configure]` call however it appears this is actually thrown in the Firebase worker thread which we don't really have access to.
